src/math/ColorManagement.js

```diff
@@ -9,6 +9,8 @@ export function SRGBToLinear( c ) {
 
 export function LinearToSRGB( c ) {
 
+	if ( c === 1 ) return 1;
```

Comment:
This isn't the proper solution. (For example, it can break if we use another transfer function or maybe another color value) The proper solution is to do rounding in `getHex` and `getStyle` -- like in https://github.com/mrdoob/three.js/pull/25832.